core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -1933,7 +1933,7 @@ class WorkspaceService(protected val userInfo: UserInfo, val dataSource: SlickDa
       val workspace = Workspace(
         namespace = workspaceRequest.namespace,
         name = workspaceRequest.name,
-        authorizationDomain = workspaceRequest.authorizationDomain,
+        authorizationDomain = workspaceRequest.authorizationDomain.getOrElse(Set.empty),
```

Comment:
For readability, conciseness, (and future flexibility in case we want to change how we handle `None`), I think it makes sense to extract `getOrElse(Set.empty)` where possible.